src/item.cpp

```diff
@@ -3373,6 +3375,20 @@ bool item::is_food() const
     return type->comestible != nullptr;
 }
 
+bool item::is_med() const
```

Comment:
`is_medicine` or `is_medication`
Preferably the latter as we should eventually migrate to a `MEDICATION